app/assets/stylesheets/earthworks.css

```diff
@@ -205,3 +205,9 @@ Uncomment lines 142-144
   color: var(--stanford-digital-blue-dark) !important;
   border-color: var(--stanford-digital-blue-dark) !important;
 }
+
+:focus-visible {
```

Comment:
Should all of this go in the component library? Purl has the similar style:
<img width="566" alt="Screenshot 2024-08-26 at 2 44 17 PM" src="https://github.com/user-attachments/assets/66d8b20d-f687-4f4d-8a0a-8a45ace3498f">

Reply:
See earthworks chat.